modules/ROOT/pages/rtm-agent-proxy-config.adoc

```diff
@@ -57,6 +57,28 @@ This example configures the agent to work with a proxy server that requires auth
 amc_setup -H myToken myMuleServer -P acme.proxy.com 443 internalAdmin Ins1d3V0icePassword
 ----
 
+== Verify That the Proxy Server Does Not Modify the Agent Certificate 
+
+To ensure that your firewall or proxy does not intercept or modify the 
+Runtime Manager agent, run the following command:
+
+[source,console,linenums]
+----
+echo -e "GET / HTTP/1.0\r\n" | openssl s_client -connect mule-manager.anypoint.mulesoft.com:443 -ign_eof
```

Comment:
Taking into consideration that 
- Old agents use `mule-manager.anypoint.mulesoft.com`
- Newer agent's use `runtime-manager.anypoint.mulesoft.com` (+1.12.0 and +2.2.0) 

I think we better use `runtime-manager.anypoint.mulesoft.com` for reference.

Run the command with either host will probe if the firewall or proxy is tampering the certificate